backend/apps/events/validators.py

```diff
@@ -0,0 +1,244 @@
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+from django.utils.timezone import make_aware
+from utils.helpers.validators import email_validation
+
+
+def time_validation(start_time_input, end_time_input, start_time=None, end_time=None):
+    """
+    Method for ensuring that any updates of time values are valid.
+    Takes in the new time inputs and potentially current values (if there are any)
+    """
+    # Make all date times aware (instead of naive)
+    if start_time_input is not None and start_time_input.tzinfo is None:
+        start_time_input = make_aware(start_time_input, is_dst=True)
+
+    if start_time is not None and start_time.tzinfo is None:
+        start_time = make_aware(start_time, is_dst=True)
```

Comment:
[`is_dst` is deprecated](https://docs.djangoproject.com/en/4.0/ref/utils/#django.utils.timezone.make_aware)